docs/components/truncate.md

```diff
@@ -0,0 +1,236 @@
+# Truncate
+
+**KTruncate** - A Kongponent that limits content to a specific number of lines and provides controls to expand or collapse it.
+
+<KCard>
+  <template v-slot:body>
+    <KTruncate>
+      <KBadge v-for="n in 25" :key="n">
+        Item {{ n }}
+      </KBadge>
+    </KTruncate>
+  </template>
+</KCard>
+
+```html
+<template>
+  <KTruncate>
+    <KBadge v-for="n in 25" :key="n">
+      Item {{ n }}
+    </KBadge>
+  </KTruncate>
+</template>
+```
+
+## Props
+
+### rows
+
+Number of rows to truncate content at. Default value is `1`.
+
+<KCard>
+  <template v-slot:body>
+    <KTruncate :rows="2">
+      <KBadge v-for="n in 25" :key="n">
+        Item {{ n }}
+      </KBadge>
+    </KTruncate>
+  </template>
+</KCard>
+
+```html
+<template>
+  <KTruncate :rows="2">
+    <KBadge v-for="n in 25" :key="n">
+      Item {{ n }}
+    </KBadge>
+  </KTruncate>
+</template>
+```
+
+### isTextContent
+
+By default the component treats anything passed through the `default` slot as collection of HTML elements. Use this prop if you want to truncate text.
+
+When this prop is set to `true`, the component applies different logic; truncation is achieved via the `-webkit-line-clamp` CSS property, rather than assessing the height of child elements to determine the row height.
```

Comment:
```suggestion
When this prop is set to `true`, the component applies different logic; truncation is achieved via the `--webkit-line-clamp` CSS property, rather than assessing the height of child elements to determine the row height.
```

Reply:
It's actually `-webkit-line-clamp`, not `--webkit-line-clamp`

Reply:
😨 whoops